media-video/ffmpeg/ffmpeg-4.2.2.recipe

```diff
@@ -63,9 +63,37 @@ REQUIRES="
 	"
 if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
 	REQUIRES+="
+		lib:libass$secondaryArchSuffix
+#		lib:libbluray$secondaryArchSuffix
```

Comment:
FWIW @extrowerk I think I had problems with ffmpeg crashes on libbluray also (didn't check on this PR for this though)